cognite/client/data_classes/transformations/__init__.py

```diff
@@ -169,6 +174,14 @@ def __init__(
         self.tags = tags
         self._cognite_client = cast("CogniteClient", cognite_client)
 
+    @property
+    def has_source_oidc_credentials(self) -> bool:
+        return self.source_oidc_credentials is not None
+
+    @property
+    def has_destination_oidc_credentials(self) -> bool:
+        return self.destination_oidc_credentials is not None
```

Comment:
If they are set to `False`, these return `True`

Reply:
What do you mean? The `self.destination_oidc_credentials` should bi set to `OidcCredentials`.

Reply:
I can't read 😂